### PR TITLE
Refer to wikipedia

### PR DESCRIPTION
**Pull request by Shynixn**

### CODE REVIEW

File: Rezept.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Gugelhupf
 
-Ein Rezept für einen Gugelhupf.
+Ein Rezept für einen [Gugelhupf](https://de.wikipedia.org/wiki/Gugelhupf).
```
Comment (meik99) [inline Rezept.md]: Please link english version of wiki site since german speaking people most likely know what a Gugelhupf is

**Review by meik99 — CHANGES_REQUESTED** [review]

File: Rezept.md (hunk 1 above)
Comment (Shynixn) [reply Rezept.md]: ok, I'll link the English wikipedia in your German text  ☕️

Comment (meik99) [reply Rezept.md]: English speaking people have as much rights to Gugelhupf as german speaking ones

**Comment by Shynixn** [comment]
Done.

**Review by meik99 — APPROVED** [review]

**Pull request merged**